.github/workflows/release_bundle.yml

```diff
@@ -68,21 +105,28 @@ jobs:
            --operation=create
           source ./result.env
           echo "$CREATED_TAG" 
-          echo "created_tag=$CREATED_TAG" >> "$GITHUB_OUTPUT"
-          echo "created_tag=$CREATED_TAG" >> $GITHUB_ENV
+          echo "created_tag=$CREATED_TAG" >> $GITHUB_OUTPUT
 
-      - name: Test install from Github Release
+      - name: "Test install from Github Release"
         uses: ./.github/actions/test_install
         with:
-          version_number: ${{ env.created_tag }}
+          version_number: ${{ steps.create-release.outputs.created_tag }}
           python_version: "3.8"
           os_platform: "linux"
 
+      - name: "Post Notification"
+        run: |
+          title="Test Install Successful"
+          message="Installation and version command run successful for os_platform=linux, Python="3.8", version=${{ steps.create-release.outputs.created_tag }}"
+          echo "::notice $title::$message"
+
   build-for-os-and-python-versions:
-    needs: create-bundle
+    needs: [create-bundle, build-python-matrix]
     strategy:
+      # run even if some fail so we get a full picture.  At thois point linux/3.8 si already released anyways so there's no reason to stop
```

Comment:
Noice! I like this spelling of "this". And "is".

Reply:
```suggestion
      # run even if some fail so we get a full picture.  At this point linux/3.8 is already released anyways so there's no reason to stop
```